src/Chapter05/Listing05.16.PassingVariablesByValue.cs

```diff
@@ -0,0 +1,32 @@
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter05.Listing05_16;
+
+using System;
+
+#region INCLUDE
+public class Program
+{
+    public static void Main()
+    {
+        // ...
+        string fullName;
+        string homeFolder = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
```

Comment:
```suggestion
        // Environment.SpecialFolder.UserProfile on different OS
        // WIN: C:\Users\intellitect
        // LIN: /home/intellitect
        // OSX: /Users/intellitect
        string homeFolder = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
```
could be fun to explain the result for each os